packages/runtime/src/lib/constants.ts

```diff
@@ -322,6 +322,19 @@ export const reOct = /^0[0-8]+$/;
  */
 export const reBase64 = /^([0-9a-zA-Z+/]{4})*(([0-9a-zA-Z+/]{2}==)|([0-9a-zA-Z+/]{3}=))?$/;
 
+/** is Base64Url regex
+ * ^                          # Start of input
+ * ([0-9a-zA-Z_-]{4})*        # Groups of 4 valid characters decode
+ *                            # to 24 bits of data for each group
+ * (                          # Either ending with:
+ *     ([0-9a-zA-Z_-]{2}==)   # two valid characters followed by ==
+ *     |                      # , or
+ *     ([0-9a-zA-Z_-]{3}=)    # three valid characters followed by =
+ * )?                         # , or nothing
+ * $                          # End of input
+ */
```

Comment:
Is the text formatting correct here?

Reply:
It is formatted like that to explain the regex step by step but maybe it is not needed.